src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/ArrayTrailingComma/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="ArrayTrailingComma">
+            <property name="alwaysDemandTrailingComma" value="true"/>
+        </module>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="ArrayTrailingCommaCheck" />
```

Comment:
Can we use `checkNameForContextStrategyByTokenOrParent` instead?

Reply:
no, but we can use ancestor node, described here https://github.com/checkstyle/patch-filters/issues/302#issuecomment-674113880